tests/ResourceSyncingTest.php

```diff
@@ -763,6 +763,45 @@ function creatingResourceInTenantDatabaseCreatesAndMapInCentralDatabase()
     expect(ResourceUser::first()->role)->toBe('commenter');
 }
 
+test('resources are synced only when sync is enabled', function () {
```

Comment:
The previous PR had two tests. Did you combine their logic in this test? To test both scenarios.

Reply:
> Did you combine their logic in this test? 

Yes.